generators/createDisplayUnitSet/templates/shared_flashtalking/script/Banner.js

```diff
@@ -16,11 +17,13 @@ export default class Banner {
     const ctaCopy = document.body.querySelector('.cta_copy');
     fitText([title, ctaCopy]);
 
-    this.domMainExit = document.body.querySelector('.mainExit');
+    this.domMainExit = this.myFT.$('.mainExit');
 
-    this.domMainExit.addEventListener('click', this.handleClick);
-    this.domMainExit.addEventListener('mouseover', this.handleRollOver);
-    this.domMainExit.addEventListener('mouseout', this.handleRollOut);
+    this.domMainExit.on('click', this.handleClick);
+    this.myFT.applyClickTag(this.domMainExit, 1);
+
+    this.myFT.on('rollover', this.handleRollOver);
+    this.myFT.on('rollout', this.handleRollOut); 
```

Comment:
This seems to be working both on local server and in FT hub, so I will update it with your suggestion. Thank you so much Paulie !

Reply:
@Syridiana I've found a solution for our problem with testing the banners locally. You can put FT in a local testing mode so that it will run locally within an iframe as well. Maybe we could add this to the constructor in Banner.js and remove the log? It's  > this.myFT.testMode
`constructor(config) {
    // add required components here
    this.config = config;
    this.myFT = new FT();
    if (DEVELOPMENT) {
      this.myFT.testMode = true; //to enable local testing
    }
  }`

Reply:
Great suggestion ! 
I've commited the update